pkg/apis/core/types.go

```diff
@@ -3508,6 +3512,11 @@ type LoadBalancerIngress struct {
 	// (typically AWS load-balancers)
 	// +optional
 	Hostname string
+
+	// Ports is a list of error records of service ports
+	// If used every port defined in the service must have an entry in it
+	// +optional
+	Ports []PortStatus
```

Comment:
Will we validate that this list is identical to service ports? That seems problematic for edits to spec.Ports, where thi would not be in sync.  Maybe change "must" to "should" in the above?

Reply:
No, I have not planned validation this far. I changed it to "should".